### PR TITLE
VACMS-17307: Update alert and link for cta widget

### PR DESCRIPTION
**Pull request by chriskim2311**
## Summary
Update Alert and link for CTA widget

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17307

## Testing done
Tested locally and in review instance

## Screenshots
VA_Prescription_Refill_And_Tracking___Veterans_Affairs

VA_Prescription_Refill_And_Tracking___Veterans_Affairs

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### CODE REVIEW

File: src/applications/static-pages/cta-widget/components/CallToActionAlert.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default function CallToActionAlert({
 }) {
   const CustomHeaderLevel = `h${headerLevel}`;
   return (
-    <va-alert visible status={status}>
+    <va-alert visible status={status} usdws>
```
Comment (jamigibbs) [inline src/applications/static-pages/cta-widget/components/CallToActionAlert.jsx:17]: I think you meant for this to be `uswds`?

Comment (chriskim2311) [reply src/applications/static-pages/cta-widget/components/CallToActionAlert.jsx]: Thank you, updated!

**Review by randimays — APPROVED** [review]
LGTM!

**Comment by laflannery** [comment]
@chriskim2311 I found 2 that are still using the v1 component, are these ours taht should be updated?
1. [Education letters](http://8161cb61369dd477e719ff91aa4a2eca.review.vetsgov-internal/education/download-letters/)
2. [Verify school enrollment](http://8161cb61369dd477e719ff91aa4a2eca.review.vetsgov-internal/education/verify-school-enrollment/)

**Comment by chriskim2311** [comment]
@laflannery Nice catch,  these are different from the CTA widget. @jilladams Do you know if we own the 2 CTAs in Laura's comment above?

**Comment by randimays** [comment]
@chriskim2311 @laflannery @jilladams I checked these - they are owned by the Education Benefits team.

**Comment by laflannery** [comment]
THanks @randimays - can we/should we update [this file here](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/static-pages/cta-widget/ctaWidgets.js#L16) in that case? Because I was working off this list for a couple things (this being one of them) and that's how I found these.

**Comment by randimays** [comment]
@laflannery This actually begs the question: **should** those two instances be using the shared CTA widget? 🤔 In terms of updating this file: https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/static-pages/cta-widget/ctaWidgets.js#L16 I'm not completely sure what it's used for. I'll have to do some investigation.

**Pull request merged**